app/models/classification.rb

```diff
@@ -111,6 +111,12 @@ def metadata
     read_attribute(:metadata).with_indifferent_access
   end
 
+  def export_row
```

Comment:
This is a hack around the has_one requiring knowledge of the `classification.project_id` to hit the specific index (project_id, classification_id), this index is required to filter the large table into smaller dump read chunks. This can probably go as it's not being used in the current setup.
```
# this can't be used in joins as the instance attributes are not known ahead of time
has_one :classification_export_row, -> (c) { (project_id: c.project_id, classification_id: c.id) }
```